README.md

```diff
@@ -2244,6 +2244,8 @@ arguments that start with the prefix `--terragrunt-`. The currently available op
   under these directories (and all dependent modules) will be included during execution of the commands. If a relative 
   path is specified, it should be relative from `--terragrunt-working-dir`. Flag can be specified multiple times.
 
+* `--terragrunt-ignore-dependency-order`: Disregard the depedency between modules when using `plan-all`.
```

Comment:
I'd vote for this to be a general purpose flag that works with any `xxx-all` command. It's not a good idea to use it with other commands most of the time, but I'm not sure we should block it.

Reply:
Done